kDrive/AppRouter.swift

```diff
@@ -464,12 +475,20 @@ public struct AppRouter: AppNavigable {
             return nil
         }
 
-        let tabBarViewController = RootSplitViewController(driveFileManager: driveFileManager, selectedIndex: selectedIndex)
-
-        window.rootViewController = tabBarViewController
-        window.makeKeyAndVisible()
-
-        return tabBarViewController
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            let rootSplitViewController = RootSplitViewController(
+                driveFileManager: driveFileManager,
+                selectedIndex: selectedIndex
+            )
+            window.rootViewController = rootSplitViewController
+            window.makeKeyAndVisible()
+            return rootSplitViewController
+        } else {
+            let tabBarViewController = MainTabViewController(driveFileManager: driveFileManager)
```

Comment:
With this change, the apps forgets the selected tab. It used to be passed as a parameter.

```suggestion
            let tabBarViewController = MainTabViewController(driveFileManager: driveFileManager, selectedIndex: selectedIndex)
```